public/app/plugins/datasource/grafana-pyroscope-datasource/pyroscopeql/pyroscopeql.ts

```diff
@@ -2,7 +2,7 @@ import type { languages } from 'monaco-editor';
 
 export const languageConfiguration: languages.LanguageConfiguration = {
   // the default separators except `@$`
-  wordPattern: /(-?\d*\.\d\w*)|([^`~!#%^&*()\-=+\[{\]}\\|;:'",.<>\/?\s]+)/g,
+  wordPattern: /(-?\d*\.\d\w*)|([^`~!#%^&*()=+\[{\]}\\|;:'",<>\/?\s]+)/g,
```

Comment:
This makes it so that we consider a `.` and a `-` to be part of a word, helping when replacing existing label values with new ones (e.g., trying to replace `us-east-1` would lead to `us-east-2-east-1`).